compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -111,9 +111,6 @@ internal class ComposeContainer(
                 error("Customizing windowContainer cannot be used with LayerType.OnSameCanvas")
             }
 
-            _windowContainer?.removeComponentListener(this)
-            value.addComponentListener(this)
```

Comment:
It's incorrect. `windowContainer` is a reference for size, not a window. We can register additional event, but not replace existing one.

Use case: From #992 but setup `windowContainer` as only part of the window. We need to improve out test cases if it's not covered yet to prevent breaking this.

Reply:
See latest commit.